website/files/models/base.py

```diff
@@ -91,6 +91,19 @@ def restore(self, recursive=True, parent=None):
         TrashedFileNode.remove_one(self)
         return restored
 
+    def get_guid(self, create=False):
+        """Attempt to find a Guid that points to this object.
+        One will be created if requested.
+        :rtype: Guid
+        """
+        try:
+            # Note sometimes multiple GUIDs can exist for
+            # a single object. Just go with the first one
+            return Guid.find(Q('referent', 'eq', self))[0]
```

Comment:
This is still using `.find()`; @TomBaxter did you mean to change this to `.find_one`? Keep in mind, you'll also need to handle `NoResultsFound` if you make that change.